src/lib/public/copy/copy.ts

```diff
@@ -0,0 +1,13 @@
+import { toast, ToastType } from '$lib/components/toaster/toasts';
+
+export default function copy(
+  link: string,
+  successMessage: string,
+  path: string,
+  errorMessage: string
```

Comment:
The original `copy` function, featured in [src/routes/(site)/events/event.svelte](https://github.com/EthanThatOneKid/acmcsuf.com/blob/933fa16bad61ef85c224950614c6d67edaa9cf6b/src/routes/(site)/events/event.svelte#L17), contained these arguments in the order:
1. link: string
2. successMessage: string
3. errorMessage: string
4. path: string

Where `path` represents an identifier for a specific team name used by the `toast` function to determine a team-specific highlight color. Acknowledging #470, we can choose the name `teamId` over `path`.

```suggestion
  link: string,
  successMessage: string,
  errorMessage: string,
  teamId: string
```

Reply:
done